### PR TITLE
scrolling by button to show album(playlist)

### PR DESCRIPTION
**Pull request by wild-flame**
add a new feature to main page, click the button to load more.
could be replaced by infinite scrolling or paginate in future implementation.

image

By the way, I'm a bit confused about the QQ api. the implementation now  seems to updated load the list of album in different order.

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
       return value && JSON.parse(value);
   }
 
-  var app = angular.module('listenone', ['angularSoundManager', 'ui-notification', 'loWebManager'])
+    var app = angular.module('listenone', ['angularSoundManager', 'ui-notification', 'loWebManager'])
```
Comment (listen1) [inline js/app.js]: No necessary

File: listen1.html [hunk 2]
```diff
@@ -20,6 +20,7 @@
     <script type="text/javascript" src="js/vendor/angular.min.js"></script>
     <script type="text/javascript" src="js/vendor/angular-soundmanager2.js"></script>
     <script type="text/javascript" src="js/vendor/angular-ui-notification.js"></script>
+    <script type="text/javascript" src="js/vendor/angular-infinite-scroll.js"></script> <!-- require jQuery as dependency -->
```
Comment (listen1) [inline listen1.html]: Not used library

**Comment by listen1** [comment]
Cool pull request! Thank you so much.
Only a few tips before merging please check them.

**Comment by wild-flame** [comment]
@listen1 fixed :D

**Pull request merged**